### DISCUSSION TITLE
How about Fresh Conference

### DISCUSSION THREAD

**Discussion by ghost** [freshframework/fresh#1049] [General]
I think it would be very exciting to have a Fresh event etc...Nextjs and Vite have conferences, why not Fresh too?

I created a template for a conference site in a few minutes (I really only made a few, so the design may not be great).

https://freshcon.deno.dev

What do you think?

**Comment by ghost** [comment]
I renewed the website: https://freshcon.deno.dev (wip)

**Reply by ghost** [reply]
And I made a repository: https://github.com/kstdx/freshcon

**Reply by hashrock** [reply]
Haha cute site. denoconf isn't even here yet, so maybe it's a bit too early😅
